tests/Unit/IO/Test_VolumeData.py

```diff
@@ -10,159 +10,148 @@
-        # source tree (for file size reasons), so it should not be used as an
-        # example output for any other reason than testing VolumeData.
-        self.file_name_r = os.path.join(Informer.unit_test_path(),
-                                        "IO/VolTestData.h5")
-        if os.path.isfile(self.file_name_w):
-            os.remove(self.file_name_w)
+        # The tests in this class involve inserting vol files, the h5 file
+        # will be deleted and recreated for each test
+        self.file_name = os.path.join(Informer.unit_test_path(),
+                                      "IO/TestVolumeDataWriting.h5")
+        if os.path.isfile(self.file_name):
+            os.remove(self.file_name)
+        self.h5_file = spectre_h5.H5File(file_name=self.file_name,
+                                         append_to_file=True)
 
     def tearDown(self):
-        if os.path.isfile(self.file_name_w):
-            os.remove(self.file_name_w)
+        self.h5_file.close()
+        if os.path.isfile(self.file_name):
+            os.remove(self.file_name)
 
     # Testing the VolumeData Insert Function
     def test_insert_vol(self):
-        h5_file = spectre_h5.H5File(file_name=self.file_name_w,
-                                    append_to_file=True)
-        h5_file.insert_vol(path="/element_data", version=0)
-        vol_file = h5_file.get_vol(path="/element_data")
+        self.h5_file.insert_vol(path="/element_data", version=0)
+        vol_file = self.h5_file.get_vol(path="/element_data")
         self.assertEqual(vol_file.get_version(), 0)
-        h5_file.close()
 
     # Test the header was generated correctly
     def test_vol_get_header(self):
-        h5_file = spectre_h5.H5File(file_name=self.file_name_w,
-                                    append_to_file=True)
-        h5_file.insert_vol(path="/element_data", version=0)
-        vol_file = h5_file.get_vol(path="/element_data")
+        self.h5_file.insert_vol(path="/element_data", version=0)
+        vol_file = self.h5_file.get_vol(path="/element_data")
         self.assertEqual(vol_file.get_header()[0:20], "#\n# File created on ")
-        h5_file.close()
 
-    # The other tests require external data, so they use the included h5 file
-    # `VolTestData.h5` which contains spectre output data (see above).
-    # Test the observation ids and values are correctly retrived
+
+class TestVolumeData(unittest.TestCase):
+    # Test Fixtures
+    def setUp(self):
+        # The tests in this class use a volume data file written using
+        # the write_volume_data() function
+        self.file_name = os.path.join(Informer.unit_test_path(),
+                                      "IO/TestVolumeData.h5")
+
+        if os.path.isfile(self.file_name):
+            os.remove(self.file_name)
+
+        self.h5_file = spectre_h5.H5File(file_name=self.file_name,
+                                         append_to_file=True)
+        self.tensor_components_and_coords = np.random.rand(4, 8)
```

Comment:
Delete `_and_coords` suffix

Reply:
What about using `tensor_component_dat`a here and `tensor_component_names` for field names. 
Currently, `tensor_component`s is used in `test_tensor_components()` for the field names.

Reply:
sounds good :)